papers/FC20/paper/appendix-governance.tex

```diff
@@ -46,14 +46,15 @@ \subsubsection{Voting Results}
 
 The sum of the private stake in favour and delegated stake in favour forms the \emph{stake in favour}; similarly, we obtain the \emph{stake against} as well as the \emph{abstaining stake}.
 
-Suppose that the \emph{honest stake threshold assumption} of our software updates protocol is $h$, then at the end of the tallying, a software update (i.e., a SIP or an UP) is marked one of the following:
+%Suppose that the \emph{honest stake threshold assumption} of our software updates protocol is $h$
+Suppose that we define a voting threshold $\tau_V$, then at the end of the tallying, a software update (i.e., a SIP or an UP) is marked one of the following:
 \begin{itemize}
 
-\item \emph{Approved}. When the $stake\ in\ favour \geq h$
+\item \emph{Approved}. When the $stake\ in\ favour > \tau_V$
```

Comment:
A minor thing, but for text inside a math environment you might want to use something like `mathit`.